framework/src/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -707,10 +707,13 @@ public AnnotationMirror getAnnotationFromJavaExpressionString(
 
         AnnotationMirror annotationMirror = null;
         if (CFAbstractStore.canInsertReceiver(expressionObj)) {
-            Value value = getStoreBefore(tree).getValue(expressionObj);
-            if (value != null) {
-                annotationMirror =
-                        AnnotationUtils.getAnnotationByClass(value.getAnnotations(), clazz);
+            Store store = getStoreBefore(tree);
```

Comment:
I had to put this back in because without it there was a crash in when building the jdk.   It seems like correct code to me -- it just needs to be in a different PR.